### PR TITLE
DEVX-56: added more 'defensive programming'

### PR DESCRIPTION
**Pull request by kostya-luxuryescape**

### CODE REVIEW

File: src/secretsMasker.ts [hunk 1]
```diff
@@ -51,9 +51,9 @@ export const secretsMasker = (additionalSecretPaths?: Array<string>, additionalS
     serialize: false,
     censor: maskText});
 
-  const cleanMessage = (message: string) : string =>  {
+  const cleanMessage = (message: string | any) : string =>  {
```
Comment (keithbro) [inline src/secretsMasker.ts:54]: In what scenario will it not be a string?

Comment (keithbro) [reply src/secretsMasker.ts:54]: never mind - i see the test

Comment (keithbro) [reply src/secretsMasker.ts:54]: Seems like this type is wrong - https://github.com/winstonjs/logform/blob/master/index.d.ts#L9 - thats disappointing.

You might as well just make the type here any as its equivalent to `string | any`

**Review by keithbro — APPROVED** [review]

File: src/secretsMasker.ts (hunk 1 above)
Comment (kostya-luxuryescape) [reply src/secretsMasker.ts:54]: apparently in this:  `logger.info({hi:"there"})`

this way winston considers the first parameter as a message.

**Comment by kostya-luxuryescape** [comment]
I left this type as (string | any) just to put an emphasise that is should be a string as I can imagine this function can be used on it's own in projects that haven't been converted to lib-logger completely.

**Pull request merged**